### ISSUE TITLE
Permissions

### ISSUE THREAD

**Issue by vgmoose** [vgmoose/wiiu-hbas#4]
There should be permission attributes that display a small icon under the app button that shows which permissions it has.

It would be configured in meta.xml (see #3) like so:

```
<permissions>
    <sd>yes</sd>
    <usb>yes</usb>
    <wifi>yes</wifi>
    <sound>yes</sound>
<permissions>
```

**Comment by CreeperMario** [comment]
Excuse me for asking, but what would these permissions actually be? Kernel access? SD Card storage? What are they?

**Comment by vgmoose** [comment]
Looks like github removed the XML formatting in my initial post... resulting in 4x "yesses". I've edited it now.

I had planned: sd, usb, wifi, and sound, just as a heads up for the user. Kernel is interesting, although if I understand correctly, anything executed from HBL requires kernel access (although the same could be said about the other permissions). 

Maybe something additional things like: memory modification or elf loading permissions.

**Comment by CreeperMario** [comment]
Yes, I believe that is correct. Because the kernel exploit has been run, any running program can access the kernel memory, its just that most software doesn't choose to (e.g. Loadiine GX2 utilises this functionality, but Space Game doesn't).

**Comment by ghost** [comment]
Great Idea !
